src/cloud-api-adaptor/install/charts/peerpods/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+name: peerpods
+description: Cloud API Adaptor (Peerpods) Helm Chart
+type: application
+version: 0.1.0
```

Comment:
Is this chart version designed to be independent of the appVersion, or is there any value in sync-ing it like we do in Kata and the coco Chart.yaml?

Reply:
Chart version should be independent of appVersion. Chart version tracks changes to the chart itself (templates, structure), while appVersion tracks which CAA version gets deployed. This lets us fix chart bugs or support different CAA versions without forcing version locksetp.

For instance: independent versioning lets us fix chart bugs without waiting for a new CAA release.